packages/api-providers/src/ext-providers/web3/web3-provider.ts

```diff
@@ -197,4 +203,11 @@ export class Web3Provider<T = unknown> {
       },
     });
   }
+
+  async sign(payload: `0x${string}`, account: string): Promise<string> {
```

Comment:
Can we move the logic of this function to the WebbWeb3Provider's `sign` function, and rename that function to `signSubstrateAddressForClaim`? The way it is now, it seems like this is supposed to be a generic signing function when it is actually very specific.

Reply:
Sure sounds good

Reply:
Made the `sign` function generic now just takes a string, the substrate claims logic now in the hooks